00-HTML-CSS-basics/Ex1/html5multimedia.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8"/>
+  <meta name="author" content="Jacobo Andrés Dominguez"/>
+  <meta name="description" content="Exercise 1 | Doctypes and Metatags"/>
+  <meta name="keywords" content="Globant, Bootcamp, HTML5"/>
+  <meta name="viewport" content="width=device-width,initial-scale=1"/>
+  <title>HTML5 multimedia</title>
+</head>
+<body>
+  <h2>Native HTML5 multimedia support</h2>
+  <h3>Image</h3>
+  <img alt="Just a simple image." src="./multimedia/img.jpg"/>
+  <h3>Video</h3>
+  <video width="400" controls>
+    <source src="./multimedia/video.mp4" type="video/mp4">
```

Comment:
This is fine, as a good practice you may need to add different kind of sources to prevent the browser to not support it. Anyway the caption is the only needed stuff to comply with compatibility. Good work